gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -323,7 +323,64 @@
               </p>
             </div>
           </div>
+
+          <div class="row">
+            <div class="col-md-12">
+              <p>
+                <label translate>Xattr Filter</label>
```

Comment:
```suggestion
                <label translate>Extended Attributes Filter</label>
```

The abbreviation isn't used anywhere else in the GUI, so I think it's better to keep things in "human" language whenever possible 😉. The advanced Docs and the code are another story, of course.